transformer_engine/pytorch/csrc/extensions/transpose.cu

```diff
@@ -335,6 +338,8 @@ at::Tensor fp8_transpose(at::Tensor input,
 
   size_t M = static_cast<size_t>(input.size(0));
   size_t N = static_cast<size_t>(input.size(1));
+  if (M == 0 || N == 0)
+    return input;
```

Comment:
@Victarry This will cause shape mismatch error between wgrad and weight when gradient accumulation fusion is disabled.